core/database.rb

```diff
@@ -15,8 +15,10 @@ module Database
     Formats
     Persons
     preferred_formats
-    RoundTypes
+    RanksSingle
+    RanksAverage
```

Comment:
Just added two tables I needed in the computation. I whitelist tables to export in order to decrease the import time.